src/components/views/dialogs/SessionRestoreErrorDialog.js

```diff
@@ -68,7 +77,7 @@ export default React.createClass({
 
                     { bugreport }
                 </div>
-                <div className="mx_Dialog_buttons">
+                <div className="mx_Dialog_buttons" autoFocus={SdkConfig.get().bug_report_endpoint_url ? false : true}>
```

Comment:
This is quite an opaque way of handling lack of bug report endpoint. I think we should be auto-focussing this button at all times. If there is a reason to focus the link to send a bug report, feel free to explain. If we really should focus the link, I'd prefer it if we used some variable here like `shouldFocusContinueButton`, which is true/false depending on the configuration.

Reply:
Ooops, the focus should be placed to the button not to its parent div. Also the div has no autoFocus so this was all wrong I think.
Hopefully it's now a bit more readable after I have created the variable you have requested.